src/index.js

```diff
@@ -69,76 +60,151 @@ class Saturn {
       clearTimeout(connectTimeout)
 
       const { headers } = res
-      log.ttfb = new Date()
+      log.ttfbMs = new Date() - log.startTime
       log.httpStatusCode = res.status
-      log.cacheStatus = headers.get('saturn-cache-status')
-      log.nodeId = headers.get('saturn-node-id') ?? headers.get('x-ipfs-pop')
+      log.cacheHit = headers.get('saturn-cache-status') === 'HIT'
+      log.nodeId = headers.get('saturn-node-id')
       log.transferId = headers.get('saturn-transfer-id')
       log.httpProtocol = headers.get('quic-status')
 
       if (!res.ok) {
-        throw new Error(`Non OK response received: ${res.status} ${res.statusText}`)
+        throw new Error(
+          `Non OK response received: ${res.status} ${res.statusText}`
+        )
       }
+    } catch (err) {
+      log.ifNetworkError = err.message
```

Comment:
what does "if network error" mean here?

Reply:
it just means a `fetch` error occurred because it only throws for network errors. The "if" prefix is a bit wonky but somebody thought it was a better naming scheme.